terraform/node_resource_apply_instance.go

```diff
@@ -39,11 +38,6 @@ var (
 	_ GraphNodeAttachDependencies = (*NodeApplyableResourceInstance)(nil)
 )
 
-// GraphNodeAttachDestroyer
-func (n *NodeApplyableResourceInstance) AttachDestroyNode(d GraphNodeDestroyerCBD) {
-	n.destroyNode = d
-}
-
 // CreateBeforeDestroy checks this nodes config status and the status af any
```

Comment:
I think this comment is out of date now? It's no longer checking the companion destroy node?